Glyssen/ReferenceText.cs

```diff
@@ -533,5 +533,15 @@ protected override string ProjectFolder
 				return m_projectFolder;
 			}
 		}
+
```

Comment:
Note: This has already been added (elsewhere in the file) in master, so this will result in a build error when merged.